commercetools-models/src/test/java/io/sphere/sdk/carts/CartLikeTest.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.DefaultCurrencyUnits.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CartLikeTest {
+
+    private Cart cartWithTaxes;
+    private Cart cartWithoutTaxes;
+
+    @Before
+    public void setUp() throws Exception {
+        cartWithTaxes = SphereJsonUtils.readObjectFromResource("cart-with-taxes.json", Cart.class);
+        cartWithoutTaxes = SphereJsonUtils.readObjectFromResource("cart-without-taxes.json", Cart.class);
+    }
+
```

Comment:
These tests are not very easy to understand and verify, since the prices are extracted from the JSON file and that file is not precisely small. I'm not sure how to do it better, because the JVM SDK as it is now and without using any mocking framework, allows only either to provide your own implementation of the interface (as in PriceUtilsTest) or to parse a JSON file like in here. In the future I would like it to provide testing doubles.

Anyway, any idea on how to make it better?